devicemodel/README.rst

```diff
@@ -3,10 +3,12 @@ ACRN Device Model
 
 Introduction
 ============
+
 The ACRN Device Model provides **device sharing** capabilities between the
-Service OS and Guest OSs. It is a component that is used in conjunction with
-the `ACRN Hypervisor`_ and this is installed within the Service OS. You can
-find out more about Project ACRN on the `Project ACRN documentation`_ website.
+Service VM and post-launched User VMs. It is a component that is used in
+conjunction with the `ACRN Hypervisor`_ and is installed within the Service
+VM. You can find out more about Project ACRN on the `Project ACRN
```

Comment:
We can leave off the "You can find more about..." since we're actually  reading that documentation :)